ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -97,6 +97,12 @@ extension BrowserViewController: WKNavigationDelegate {
         hideReaderModeBar(animated: false)
       }
     }
+
+    // Add the current WebView URL to the redirect chain because this is the
+    // redirect source URL.
+    if let tab = tab(for: webView), let url = webView.url, rewards.isEnabled {
+      tab.redirectURLs.append(url)
```

Comment:
@aseren can you please confirm the order of the redirect chain is the same as desktop, by posting both desktop and iOS? Thanks

Reply:
tested with https://www.superuser.com/q/1471861/ url. Desktop and  iOS has the same redirect chain:
```
https://www.superuser.com/q/1471861/, 
https://www.superuser.com/questions/1471861/website-with-multiple-redirects,
https://superuser.com/questions/1471861/website-with-multiple-redirects
```